aesara/link/numba/dispatch.py

```diff
@@ -1334,3 +1410,400 @@ def broadcastto(x, *shape):
+        for m in np.ndindex(res.shape):
+            v = fn(x_reaxis[m])
+            res[m] = v
+        return res
+
+    return axis_apply_fn
+
+
+@numba_funcify.register(MaxAndArgmax)
+def numba_funcify_MaxAndArgmax(op, node, **kwargs):
+    axis = op.axis
+    x_at = node.inputs[0]
+    x_dtype = x_at.type.numpy_dtype
+    x_dtype = numba.np.numpy_support.from_dtype(x_dtype)
+    x_ndim = x_at.ndim
+
+    if x_ndim == 0:
+
+        @numba.njit(inline="always")
+        def maxandargmax(x):
+            return x, 0
+
+    else:
+
+        if axis is None:
+            axes = tuple(range(x_ndim))
+        else:
+            axes = tuple(int(ax) for ax in axis)
+
+        # NumPy does not support multiple axes for argmax; this is a
+        # work-around
+        keep_axes = tuple(i for i in range(x_ndim) if i not in axes)
+
+        reduce_max = numba.njit(
+            create_multiaxis_reducer(np.maximum, -np.inf, axes, x_ndim, x_dtype)
+        )
+        reduced_x_ndim = x_ndim - len(axes) + 1
+        argmax_axis = create_axis_apply_fn(
+            np.argmax, reduced_x_ndim - 1, reduced_x_ndim, np.int64
+        )
+
+        reaxis_order = keep_axes + axes
+        sl1 = slice(None, len(keep_axes))
+        sl2 = slice(len(keep_axes), None)
+
+        @numba.njit
+        def maxandargmax(x):
+            max_res = reduce_max(x)
+
+            # Not-reduced axes in front
+            transposed_x = np.ascontiguousarray(np.transpose(x, reaxis_order))
+            kept_shape = transposed_x.shape[sl1]
+            reduced_shape = transposed_x.shape[sl2]
+            reduced_size = 1
+            for s in reduced_shape:
+                reduced_size *= s
+
+            # Numpy.prod returns 1.0 when arg is empty, so we cast it to int64
+            # Otherwise reshape would complain citing float arg
+            new_shape = kept_shape + (reduced_size,)
+            reshaped_x = transposed_x.reshape(new_shape)
+
+            max_idx_res = argmax_axis(reshaped_x)
+
+            return max_res, max_idx_res
+
+    return maxandargmax
+
+
+@numba_funcify.register(Cholesky)
+def numba_funcify_Cholesky(op, node, **kwargs):
+    lower = op.lower
+
+    out_dtype = node.outputs[0].type.numpy_dtype
+
+    if lower:
+
+        inputs_cast = int_to_float_fn(node.inputs, out_dtype)
+
+        @numba.njit
+        def cholesky(a):
+            return np.linalg.cholesky(inputs_cast(a)).astype(out_dtype)
+
+    else:
+        # TODO: Use SciPy's BLAS/LAPACK Cython wrappers.
+
+        warnings.warn(
+            (
+                "Numba will use object mode to allow the "
+                "`lower` argument to `scipy.linalg.cholesky`."
+            ),
+            UserWarning,
+        )
+
+        ret_sig = get_numba_type(node.outputs[0].type)
+
+        @numba.njit
+        def cholesky(a):
+            with numba.objmode(ret=ret_sig):
+                ret = scipy.linalg.cholesky(a, lower=lower).astype(out_dtype)
```

Comment:
Isn't this available through `numba-scipy`?

Reply:
No, right now, `numba-scipy` only exposes the Cython signatures exposed by `scipy.special.cython_special`.